react-app/src/components/map/sidebar/MapSidebar.tsx

```diff
@@ -0,0 +1,189 @@
+  if (map.current) {
+    map.current.addEventListener('zoomend', definePropertiesInBounds);
+    map.current.addEventListener('moveend', definePropertiesInBounds);
+  }
+
+  return (
+    <>
+      <Box
+        id="map-sidebar"
+        zIndex={1000}
+        position={'fixed'}
+        right={open ? 0 : '-400px'}
+        height={'90%'}
+        component={Paper}
+        width={'350px'}
+        overflow={'hidden'}
+        sx={{
+          transition: 'ease-in-out 0.5s',
+        }}
+      >
+        {/* Sidebar Header */}
+        <Grid container height={50} sx={{ backgroundColor: 'rgb(221,221,221)' }}>
+          <Grid item xs={2} display={'flex'} justifyContent={'center'} alignItems={'center'}>
+            <IconButton>
+              <FilterList />
+            </IconButton>
+          </Grid>
+          <Grid item xs={8} display={'flex'} justifyContent={'center'} alignItems={'center'}>
+            <IconButton
+              size="small"
+              onClick={() => {
+                if (pageIndex > 0) {
+                  setPageIndex(pageIndex - 1);
+                }
+              }}
+            >
+              <ArrowCircleLeft fontSize="small" />
+            </IconButton>
+            <Typography
+              margin={'0 0.5em'}
+              fontSize={'0.8em'}
+            >{`${pageIndex + 1} of ${formatNumber(Math.ceil(propertiesInBounds.length / propertyPageSize))} (${formatNumber(propertiesInBounds.length)} items)`}</Typography>
+            <IconButton
+              size="small"
+              onClick={() => {
+                if (pageIndex + 1 < Math.ceil(propertiesInBounds.length / propertyPageSize)) {
+                  setPageIndex(pageIndex + 1);
+                }
+              }}
+            >
+              <ArrowCircleRight fontSize="small" />
+            </IconButton>
+          </Grid>
+          <Grid item xs={2} display={'flex'} justifyContent={'center'} alignItems={'center'}>
+            <IconButton onClick={() => setOpen(false)}>
+              <Icon sx={{ mb: '2px' }}>
+                <img height={18} width={18} src={sideBarIcon} />
+              </Icon>
+            </IconButton>
+          </Grid>
+        </Grid>
+
+        {/* List of Properties */}
+        <Box overflow={'scroll'} height={'95%'}>
+          {propertiesInBounds
+            .slice(pageIndex * propertyPageSize, pageIndex * propertyPageSize + propertyPageSize)
+            .map((property) => (
+              <PropertyRow
+                key={`${property.properties.PropertyTypeId === PropertyTypes.BUILDING ? 'Building' : 'Land'}-${property.properties.Id}`}
+                id={property.properties.Id}
+                propertyTypeId={property.properties.PropertyTypeId}
+                classificationId={property.properties.ClassificationId}
+                title={
+                  property.properties.PropertyTypeId === PropertyTypes.BUILDING
+                    ? property.properties.Name
```

Comment:
It's unfortunate the name fields are so inconsistent. Combined with the minimal design, it's tough to know what the fields are at times. Even more when they change between parcel and building. Agree that maybe it should involve something else. Maybe PID is included as another row? Otherwise it's not possible to differentiate multiple buildings on the same parcel.

Also examples like this, where people have written a novel for the name:
![image](https://github.com/bcgov/PIMS/assets/37922247/962b8766-7027-43d7-90ee-d7f41de4794a)